packages/sdk/contractkit/src/wrappers/FederatedAttestations.ts

```diff
@@ -1,8 +1,164 @@
+    0: string
+    1: string
+    2: string
+    3: string
+  }> = proxyCall(this.contract.methods.identifierToAttestations)
+
+  /**
+   * @notice Registers an attestation directly from the issuer
+   * @param identifier Hash of the identifier to be attested
+   * @param account Address of the account being mapped to the identifier
+   * @param issuedOn Time at which the issuer issued the attestation in Unix time
+   * @dev Attestation signer and issuer in storage is set to msg.sender
+   * @dev Throws if an attestation with the same (identifier, issuer, account) already exists
+   */
+  registerAttestationAsIssuer: (
+    identifier: string | number[],
+    account: string,
+    issuedOn: number | string
+  ) => Promise<void> = proxySend(this.connection, this.contract.methods.registerAttestationAsIssuer)
+
+  /**
+   * @notice Registers an attestation with a valid signature
+   * @param identifier Hash of the identifier to be attested
+   * @param issuer Address of the attestation issuer
+   * @param account Address of the account being mapped to the identifier
+   * @param issuedOn Time at which the issuer issued the attestation in Unix time
+   * @param signer Address of the signer of the attestation
+   * @param v The recovery id of the incoming ECDSA signature
+   * @param r Output value r of the ECDSA signature
+   * @param s Output value s of the ECDSA signature
+   * @dev Throws if an attestation with the same (identifier, issuer, account) already exists
+   */
+  registerAttestation: (
+    identifier: string | number[],
+    issuer: string,
+    account: string,
+    signer: string,
+    issuedOn: number | string,
+    v: number | string,
+    r: string | number[],
+    s: string | number[]
+  ) => Promise<void> = proxySend(this.connection, this.contract.methods.registerAttestation)
+
+  /**
+   * @notice Revokes an attestation
+   * @param identifier Hash of the identifier to be revoked
+   * @param issuer Address of the attestation issuer
+   * @param account Address of the account mapped to the identifier
+   * @dev Throws if sender is not the issuer, signer, or account
+   */
+  revokedAttestations: (
+    identifier: string | number[],
+    issuer: string,
+    account: string
+  ) => Promise<void> = proxySend(this.connection, this.contract.methods.revokeAttestation)
+
+  /**
+   * @notice Revokes attestations [identifiers <-> accounts] from issuer
+   * @param issuer Address of the issuer of all attestations to be revoked
+   * @param identifiers Hash of the identifiers
+   * @param accounts Addresses of the accounts mapped to the identifiers
+   *   at the same indices
+   * @dev Throws if the number of identifiers and accounts is not the same
+   * @dev Throws if sender is not the issuer or currently registered signer of issuer
+   * @dev Throws if an attestation is not found for identifiers[i] <-> accounts[i]
+   */
+  batchRevokeAttestations: (
+    issuer: string,
+    identifiers: (string | number[])[],
```

Comment:
for more complex Array types the `Array<string | number[]>` syntax is recommended 

id  probably even do something like `type Identifier = string | number[]`  `identifiers: Array<Identifier>`